docs/cli.rst

```diff
@@ -157,6 +157,11 @@ These options must be defined before the mandatory arguments.
 ``--output``, ``-o``
     Output format. Defaults to a custom format. Can also be ``yaml`` or ``json``.
 
+    **Notice:**
+
+        The `PyYAML package <https://pypi.org/project/PyYAML/>`_ is required to use the yaml output option.
+        You need to install it separately using ``pip install PyYAML``
```

Comment:
What about us also catching the `ImportError` in the import statement in `cli.py`, and printing how to install PyYAML there too?